bazel/target_recipes.bzl

```diff
@@ -6,6 +6,7 @@ TARGET_RECIPES = {
     "benchmark": "benchmark",
     "event": "libevent",
     "tcmalloc_and_profiler": "gperftools",
+    "tcmalloc_debug": "gperftools",
```

Comment:
This is going to need some special attention in https://github.com/envoyproxy/envoy/pull/5218. Probably non-actionable for this review though.

Reply:
ack